src/themes/hugo-theme-learn/assets/sass/_prestashop.scss

```
@@ -143,6 +143,11 @@ a[data-featherlight] {
     right: 3rem;
   }
 
+  &:after {
```

```suggestion
  &::after {
```

It's a good practice in order to make it more understandable introduced in CSS3, so we distinguish Pseudo-class and pseudo-elements